civis/ml/_model.py

```diff
@@ -665,7 +665,9 @@ def from_existing(cls, train_job_id, train_run_id='latest', client=None):
             raise
 
         args = container.arguments
-        model = args['MODEL']
+
+        # Older templates used "WORKFLOW" instead of "MODEL"
+        model = args.get('MODEL', args.get('WORKFLOW'))
```

Comment:
We have only given this template these two arguments over our codebases development history.